.github/ISSUE_TEMPLATE/security_report.md

```diff
@@ -4,6 +4,4 @@ about: Do not create an issue for security reports, send an email to security@el
 
 ---
 
-Do not create an issue for security reports, send an email to:
-
-security@electronjs.org
+**DO NOT** create an issue for security reports, send an email to: [security@electronjs.org](mailto:security@electronjs.org)
```

Comment:
I don't think the markdown formatting improves things here, as the issue template will be shown in plain text.

Reply:
i'll remove the mailto!